command/sync.go

```diff
@@ -261,7 +261,11 @@ func compareObjects(sourceObjects, destObjects chan *storage.Object) (chan *url.
 
 		for {
 			if srcOk {
-				srcName = filepath.ToSlash(src.URL.Relative())
+				if isSrcBatch {
+					srcName = filepath.ToSlash(src.URL.Relative())
+				} else {
+					srcName = src.URL.Base()
+				}
```

Comment:
```suggestion
				srcName = src.URL.Base()

				if isSrcBatch {
					srcName = filepath.ToSlash(src.URL.Relative())
				}
```

or

```suggestion
				srcName = filepath.ToSlash(src.URL.Relative())

				if !isSrcBatch {
					srcName = src.URL.Base()
				}
```


could be better.